nemo/collections/common/tokenizers/code_spec.py

```diff
@@ -0,0 +1,44 @@
+from typing import List, Tuple
+
+from numpy import ndarray
+
+__all__ = ["Code"]
+
+
+class Code(object):
```

Comment:
you define the spec, where do you use it?

Reply:
In NumericCode and CategoryCode.